lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,45 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const ALLOWED_IMAGE = /^\s*url\("([^"]+)"\)\s*$/;
```

Comment:
lets do a `trim()` on the string before running this regex so we can drop the `\s*` on the ends